addons/binding/org.openhab.binding.jenocean/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="enocean"
+				 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				 xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+				 xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>EnOcean Binding</name>
+	<description>This is a binding for EnOcean.</description>
+	<author>Jan Kemmler</author>
+	<config-description-ref uri="binding:enocean:bindingUsbPort"/>
```

Comment:
You should define a bridge for the USB dongle (see e.g. ZWave & Zigbee bindings) - it must not be a binding configuration. Having it as a bridge potentially allows using multiple dongles at the same time.